setup.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS responses (
+  id    SERIAL PRIMARY KEY,
+  added_date    timestamp WITH time zone,
+  survey_response    jsonb
+);
```

Comment:
This is a sample document:
```javascript
{
  "type": "uk.gov.ons.edc.eq:surveyresponse",
  "origin": "uk.gov.ons.edc.eq",
  "survey_id": "194825",
  "version": "0.0.1",
  "collection": {
    "exercise_sid": "hfjdskf",
    "instrument_id": "10",
    "period": "0616"
  },
  "submitted_at": "2016-03-12T10:39:40Z",
  "metadata": {
    "user_id": "789473423",
    "ru_ref": "1234570071A"
  },
  "data": {
    "1": "2",
    "2": "4",
    "3": "2",
    "4": "Y"
  }
}
```

At the moment we support querying:
- "id" (primary key) and "added_date" columns,
- "survey_id", "survey_response.metadata.ru_ref" and "survey_response.collection.period" fields,

We can also query "survey_response.form" field but I haven't seen this field in test data.